kyo-data/shared/src/main/scala/kyo/Maybe.scala

```diff
@@ -16,7 +17,9 @@ export Maybe.Present
 /** Companion object for Maybe type */
 object Maybe:
     inline given [A, B](using inline ce: CanEqual[A, B]): CanEqual[Maybe[A], Maybe[B]] = CanEqual.derived
-    given [A]: Conversion[Maybe[A], IterableOnce[A]]                                   = _.iterator
+    implicit def toIterableOnce[A](v: Maybe[A]): IterableOnce[A]                       = v.iterator
```

Comment:
I've changed to `implicit def` because `Conversion` requires the user to add `import scala.language.implicitConversions`,  which isn't ideal